test/jasmine/tests/sunburst_test.js

```diff
@@ -507,7 +507,7 @@ describe('Test sunburst hover:', function() {
                 curveNumber: 0,
                 pointNumber: 0,
                 label: 'Eve',
-                parent: ''
+                parent: '"root"'
```

Comment:
This doesn't look right. `parents[0]` is `''` in the input trace, so I think we should honour that in the event data.

Reply:
OK if we return an empty string here?
https://github.com/plotly/plotly.js/blob/217c0f43020f9f13d4361aadc6cd2d1a93c020ae/src/traces/sunburst/helpers.js#L154

Reply:
What are the side-effects?

Reply:
Simply getting blank string (nothing) referring to the dummy root label in hover; which I think is fine.

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/4185/commits/8416883be97b77bdf685ade8790544e4d3b0fa8a.